src/triggeredByBot.test.ts

```diff
@@ -0,0 +1,110 @@
+import { mocked } from 'ts-jest/utils'
+import { github, PullsGetResponse } from './infrastructure/github'
+import { BOT_USERS, triggeredByBot } from './triggeredByBot'
+
+const mockGithub = mocked(github, true)
+
+jest.mock('./infrastructure/github')
+
+describe('triggeredByBot', () => {
+  describe('pul_request event', () => {
+    const pullRequestResponse = {
+      data: {
+        title: '',
+        head: {
+          ref: '',
+        },
+        user: {
+          login: '',
+        },
+      },
+    }
+
+    beforeEach(() => {
+      mockGithub.context.eventName = 'pull_request'
+      mockGithub.context.payload.pull_request = {
+        number: 5,
+      }
+
+      mockGithub.getPullRequest.mockResolvedValue(
+        pullRequestResponse as PullsGetResponse
+      )
+    })
+
+    it.each(BOT_USERS)('returns true for bot users [%s]', async (botUser) => {
+      pullRequestResponse.data.user.login = botUser
+
+      await expect(triggeredByBot()).resolves.toBeTruthy()
+    })
+  })
+
+  describe('push event', () => {
+    beforeEach(() => {
+      mockGithub.context.eventName = 'push'
+    })
+
+    it.each(BOT_USERS)(
+      'returns true if all commits have author.name of a bot user [%s]',
```

Comment:
If I merge, will the bot be put in the PR, or myself? 
I'm asking because I have merged / squashed many times and I have seen myself swell in the list

Reply:
This one is for the `push` event. I thought it would be safer to only skip if all commits are coming from a bot user, since when merging something maybe you had some changes pushed to the branch of the bot PR. But thinking about our specific issue, we technically don't need to skip the push event ever, because we'll always have the secrets available.

What are your thoughts? Should we keep the skipping only for PRs?

Reply:
I'm looking at a push like this: https://github.com/Typeform/standards-adoption-tool/commit/1ec480d96ad515da958c04f9d006004f2b18fd0c and wonder if the current implementation will trigger or not

Reply:
Mmm, good question. I think it would based on the main author (as opposed to co-authors). It's not very clear from the UI, but in the CLI we can see the author is "Typeform Security Bot" with `git show 1ec480d96ad515da958c04f9d006004f2b18fd0c`